### PR TITLE
VAGOV-910: Fixed sidebar templates and queries

### PR DESCRIPTION
**Pull request by acabouet**
## Description
This is a fixed PR for pulling sidebar navigation content from Drupal.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by acabouet** [comment]
@jbalboni see the build failing on this but I don't have jenkins access to see why. I think it's probably due to an error I'm getting with the build locally, that seems to be related to one of the liquid templates based on the trace. Error -  TypeError: Cannot read property 'offset' of undefined

**Comment by jbalboni** [comment]
@acabouet Looks like linter errors:

screen shot 2019-02-01 at 9 35 06 am

If you have Prettier set up in your editor, running it on those files will fix the lint errors. You can also run `yarn lint:js:fix` to fix them.

We have a pre-commit hook that runs our linter on files that have been changed, which you can install with `yarn prebuild` (I'm guessing you haven't ever needed to run `yarn build`, where it would get installed automatically.

File: src/site/stages/build/drupal/metalsmith-drupal.js [hunk 1]
```diff
@@ -74,6 +77,21 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
...
 
+  for (const navItem of taxonomies) {
+    const { entityBundle, name } = navItem;
+
+    if (name === 'Health Care') {
+      navItems[`${entityBundle}`] = navItem;
+    }
+  }
+
+  files['drupal/sidebar_navigation/index.html'] = {
```
Comment (jbalboni) [inline src/site/stages/build/drupal/metalsmith-drupal.js]: This looks like debugging info for the Health Care sidebar items. Is that correct?

**Review by jbalboni — APPROVED** [review]
This looks good to me, just need to fix the build errors.

File: src/site/stages/build/drupal/metalsmith-drupal.js (hunk 1 above)
Comment (acabouet) [reply src/site/stages/build/drupal/metalsmith-drupal.js]: Yup correct, to print them to the console the same way y'all did for items on pages.

**Pull request closed without merging**